language/Portuguese_Brazil/strings.po

```diff
@@ -1599,7 +1599,7 @@ msgstr "Controle da Fita"
 
 #: src/frontend/mame/ui/mainmenu.cpp:102
 msgid "BIOS Selection"
-msgstr ""
+msgstr "Seleção da BIOS"
```

Comment:
"Seleção da BIOS" means "Selection of the BIOS" while I think it would be better to say here "Seleção de BIOS" meaning "BIOS selection" or (implicitely) "Selection of a BIOS".